src/scrapers/leumi-card.js

```diff
@@ -66,6 +67,7 @@ function getTransactionType(txnTypeStr) {
     case MONTHLY_CHARGE_PLUS_INTEREST_TYPE_NAME:
       return NORMAL_TXN_TYPE;
     case INSTALLMENTS_TYPE_NAME:
+    case CREDIT_TYPE_NAME:
```

Comment:
Hi, Thanks for your contribution. I just want to confirm that קרדיט should be treated as installments (תשלומים)? isn't it a one time payment? If it is then it should result with `NORMAL_TXN_TYPE`. wdyt?